lib/src/beamer.dart

```diff
@@ -127,4 +122,24 @@ extension UpdateCurrentLocation on BuildContext {
       stacked: stacked,
     );
   }
+
+  /// See [BeamerRouterDelegate.currentLocation]
+  BeamLocation get currentBeamLocation {
```

Comment:
would it be nicer to use short syntax (`=>`) for those one-liner getters?

Reply:
I also thought about that. If you like the short syntax more I can change it :)

Reply:
It would look like this:

```
/// See [BeamerRouterDelegate.currentLocation]
BeamLocation get currentBeamLocation => Beamer.of(this).currentLocation;

/// See [BeamerRouterDelegate.currentPages]
List<BeamPage> get currentBeamPages => Beamer.of(this).currentPages;

/// See [BeamerRouterDelegate.canBeamBack]
bool get canBeamBack => Beamer.of(this).canBeamBack;

/// See [BeamerRouterDelegate.beamBackLocation]
BeamLocation get beamBackLocation => Beamer.of(this).beamBackLocation;```

Reply:
It's kind of a tough choice, but I say yes :slightly_smiling_face:

Reply:
Done :) It's definitely more clean.